packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1858,16 +1858,6 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
     if (!animation!.isCompleted) {
       return false;
     }
-    // If we're being popped into, we also cannot be swiped until the pop above
-    // it completes. This translates to our secondary animation being
-    // dismissed.
-    if (!secondaryAnimation!.isDismissed) {
-      return false;
-    }
-    // If we're in a gesture already, we cannot start another.
-    if (popGestureInProgress) {
-      return false;
-    }
```

Comment:
These seemed to be an incorrect assumption. In my testing I wanted to do two back gestures quickly back-to-back, and this code prevented that from working.

I added a test that does two backs quickly now.

TODO: See if any other tests are broken by this change..